mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -58,10 +60,19 @@ case class ScalaPresentationCompiler(
       using ec
     )
 
+  private def removeDoubleOptions(options: List[String]): List[String] =
+    options match
+      case head :: _ :: tail if forbiddenDoubleOptions(head) =>
+        removeDoubleOptions(tail)
+      case head :: tail => head :: removeDoubleOptions(tail)
+      case Nil => options
```

Comment:
Why not

```suggestion
      options.filterNot(forbiddenDoubleOptions)
```

?

Reply:
Ahh, nevermind, `removeDoubleOptions` transforms `List("-release", "8", ...)` to `List(...)`

Reply:
I had to get a bit more creative here :sweat_smile: